python/tskit/tables.py

```diff
@@ -2852,55 +2866,103 @@ def individuals(self) -> IndividualTable:
         """
         return IndividualTable(ll_table=self._ll_tables.individuals)
 
+    @individuals.setter
+    def individuals(self, value):
+        raise AttributeError(
+            "Cannot set tables in a table collection: use table.replace_with() instead."
```

Comment:
Instead of repeating this string can you make it a class attribute?

Reply:
Good idea!